.prettierignore

```diff
@@ -1,9 +1,7 @@
 out/
 dist/
-node_modules/
 npm-debug.log
 yarn-error.log
-app/node_modules/
```

Comment:
I'm intrigued that we don't need to do this any more, as I thought it'd get caught in this rule: `./app/**/*.{scss,ts,tsx,js,jsx,html}\`. Did Prettier just get smart enough to skip `node_modules`?

Reply:
yup, exactly! @j-f1 told me :)